src/APIFunctions/2DPrinting.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+
+export const range = (start, end) => {
+  const length = end - start
+  return Array.from({ length }, (_, i) => start + i)
+}
+
+export function parseRange (pages, maxPages) {
+  var result = new Set()
+  var subr = pages.split(',')
+  subr.forEach(element => {
+    const x = element.split('-')
+    const arr = range(Number(x[0]), Number(x[x.length - 1]) + 1)
```

Comment:
We gotta move away from using variable names like x or arr. Can you change `x` to something like `pageRangeSplitByDashes` and `arr` to `pagesFromDashSplit`?